src/data/ability.ts

```diff
@@ -4872,6 +4908,11 @@ export function applyPostFaintAbAttrs(attrType: Constructor<PostFaintAbAttr>,
   return applyAbAttrsInternal<PostFaintAbAttr>(attrType, pokemon, (attr, passive) => attr.applyPostFaint(pokemon, passive, simulated, attacker, move, hitResult, args), args, false, simulated);
 }
 
+export function applyPostItemLostAbAttrs(attrType: Constructor<PostItemLostAbAttr>,
+  pokemon: Pokemon, passive: boolean, simulated: boolean = false, args: any[]): Promise<void> {
```

Comment:
```suggestion
  pokemon: Pokemon, passive: boolean, simulated: boolean = false, ...args: any[]): Promise<void> {
```
Matches other `applyAbAttrs` functions. This also means you don't have to pass empty arrays as arguments (e.g. in the new code in `battle-scene.ts`)